pkg/cmd/grafana-server/server.go

```diff
@@ -163,7 +163,9 @@ func (g *GrafanaServerImpl) Run() error {
 		})
 	}
 
-	sendSystemdNotification("READY=1")
+	if err := sendSystemdNotification("READY=1"); err != nil {
+		return err
```

Comment:
What happens with the `childRoutines` when error is returned?

Reply:
@marefr Good point, thanks. I tried to fix this now by `defer`-ing the call to a function that waits on `childRoutines` and also bubbles an eventual error from the wait.